cmake/configs/default.cmake

```diff
@@ -445,7 +445,7 @@ hunter_default_version(pugixml VERSION 1.10-p0)
 hunter_default_version(pybind11 VERSION 2.5.0)
 hunter_default_version(qhull VERSION 7.2.0-p1)
 hunter_default_version(quickjs VERSION 2020-04-12-p0)
-hunter_default_version(rabbitmq-c VERSION 0.10.0)
+hunter_default_version(rabbitmq-c VERSION 0.10.0-773b883-t1)
```

Comment:
`-t1` indicates testing. Since the package works without changes from upstream, just drop the `-t1` here.